### ISSUE TITLE
Agent Issue

### ISSUE THREAD

**Issue by ma-karai** [Inlustra/auto-docker-dash#39]
Hey Ho, Still digging this project. I tried to set up an agent via ENV var and i am gettin this error

` error:   ┗ [1] { component: 'agent', error: 'Only absolute URLs are supported' }`

docker compose

```
 plugsy-agent:
    container_name: plugsy-agent
    image: plugsy/agent
    environment:
      - PLUGSY_AGENT_ENDPOINT="http://192.168.0.40:3000/graphql"
    volumes:
      - /var/run/docker.sock:/var/run/docker.sock
```

**Comment by ma-karai** [comment]
@Inlustra Hey Thomas, are you still working on plugsy ? IMHO one of the best things i have installed for my internal servers :)

**Comment by Inlustra** [comment]
Hi @ma-karai , I am still working on it, it's just been difficult to find time!

With regards to your question: you should be able to run an agent and a server on the same machine, though I would question why you would want that?

**Comment by ma-karai** [comment]
Hey @Inlustra 

The reasoning for me is just out of convenience.
I have 4 running servers and each of them has its own docker network, I did not create a large swarm network, because the swarm networks are apparently phased out. Each of the server has a single dashboard, sometimes I switch between these servers just from the plugsy plugin. Then i have a simple tablet in the living room where i would like to display all of my services and see at a glance what's online and what's not.

![image](https://user-images.githubusercontent.com/17119942/142581206-68824b99-3002-4dff-bf23-71daea0a091f.png)
![image](https://user-images.githubusercontent.com/17119942/142581343-5775b7ca-5799-4d6a-a2c6-67da4b74f89b.png)
![image](https://user-images.githubusercontent.com/17119942/142581412-7595426d-8019-4741-912a-59b0fd5c46e1.png)

Some pictures. Anyway, when i create a docker compose file with a server and an agent i can't really send the info to a different server, not sure why.

As a side question, have the labels also changed into plugsy_ instead of dockerdash ?

Also let me rephrase how i think your add-on is just so much better than all these hard coded start pages, sure having the option to see the weather outside is great and all, but for me not really useful :) So thanks again for this tool, if you pm me PayPal address I'd like to donate you some cash for dinner or a six-pack of beers or a pound of coffee. haha

**Comment by Inlustra** [comment]
@ma-karai The labels did change in the latest release: 
https://github.com/plugsy/core/releases/tag/v6.0.0

I think I should rephrase what I meant - you can configure multiple docker connectors on the same instance without needing an agent + server

An example config: 
```jsonc
{
  "$schema": "https://github.com/plugsy/core/releases/download/v6.0.0/core-config-schema.json",
  "loggingLevel": "verbose",
  "connectors": [
    {
      "type": "DOCKER",
      "config": {
        "id": "FirstMachine" // Important to use multiple ids if you have lots of the same connector
      } // This connector uses the default config (IE: Localhost)
    },
    {
      "type": "DOCKER",
      "config": {
        // This connector will connect to a different host
        "id": "SecondMachine",
        "dockerOptions": {
          // This is just an example, use an IDE and this will be autocompleted for you
          "host": "",
          "port": ""
        }
      }
    },
    {
      "type": "DOCKER",
      "config": {
        // This connector will use the default connection parameters but uses the containerMap
        // as well as the labels to grab the container
        "id": "ThirdMachine",
        "containerMap": {
          "dockerdash-core": {
            "category": "Home",
            "icon": "@styled-icons/boxicons-regular/Crown",
            "name": "Plugsy"
          }
        }
      }
    }
  ]
}
```

**Comment by ma-karai** [comment]
I figured out how to do what i intended to do, and it works great. So i would not mind sending you a sixpack of beer (if you drink) or a pizza :) up to you :)
